app/controllers/api/api_controller.rb

```diff
@@ -47,12 +50,10 @@ def check_for_auth_token
       grant = AccessGrant.find_by_access_token(token)
 
       if !grant
-        error('Cannot find AccessGrant for token #{token}')
-        return
+        return error('Cannot find AccessGrant for token #{token}')
```

Comment:
While you are here could you fix this PT story I added a while about as tech-debt?  https://www.pivotaltracker.com/story/show/156618887

The single quotes need to change to double quotes so the interpolated `#{token}` is processed.

Reply:
Sorry Github didn't show this comment to me until after I merged it.